docs/examples/writing_drivers/Instruments.ipynb

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
need to refer to this notebook now in 15 mins to qcodes, now that user/intro is gone